Themes/blur/blur.css

```diff
@@ -0,0 +1,131 @@
+/* NAME Blur **/
+/* VERSION 1.0.0 **/
+/* DESCRIPTION A simple, clean Tumblr dash theme with a soft, blurred background. **/
+/* DEVELOPER HeyItsJono - http://kaywhatever.tumblr.com/ **/
+
+body {
+	background: url('https://raw.githubusercontent.com/HeyItsJono/XKit/HeyItsJono-blur-theme/Themes/blur/body-background.png') center #000000 repeat fixed !important;
+	background-size: cover;
+}
+
+#content {
+	background: rgba(155,90,120,0.9);
+}
+
+.l-header-container--refresh {
+    background-color: rgba(50,53,60,0.67);
+}
+
+.selected .tab_notice {
+    background-color: rgb(193,33,91);
+    background: rgb(193,33,91);
+    color: rgb(193,33,91);
+}
+
+.tab-bar-container .tab_notice {
+    background-color: rgb(193,33,91);
+    background: rgb(193,33,91);
+    color: rgb(193,33,91);
+}
+
+.tab-notice--outlined {
+    border-color: rgb(193,33,91);
+}
+
+.tab_notice .tab_notice_value {
+    color: rgb(65, 66, 68);
+}
+
+.compose-button {
+    fill: rgba(200, 203, 208, 0.7);
+}
+
+.l-container.l-container--two-column .right_column, .l-container.l-container--two-column-dashboard .right_column {
+    background-color: rgba(1,1,1,0);
+}
+
+.identity .controls_section li:not(.section_header):not(.selected_blog):hover {
+    background-color: rgba(50,53,60,0.15);
+}
+
+.identity-refresh .controls_section.user_list li .follow_list_item_blog:before {
+    background-color: rgba(0, 0, 0, 0);
+    background-image: none;
+    border-right: none;
+}
+
+.identity-refresh .controls_section.user_list li:hover .follow_list_item_blog:before {
+    background: none;
+    background-image: none;
+    background-color: none;
+    border-right: none;
+}
+
+.plus-follow-button {
+    color: #414244;
+}
+
+.plus-follow-button:hover {
+    background: #C12C5B;
+}
+.identity .controls_section .item:hover .follow:hover {
+    background: #C12C5B;
+}
+
+.identity-refresh .radar .radar_footer .radar_avatar:before {
+    background-image: none;
+    background: rgba(0, 0, 0, 0);
+    background-color: rgba(0, 0, 0, 0);
+}
+
+.radar .radar_footer .radar_avatar:before {
+    background: rgba(0, 0, 0, 0);
+    background-color: rgba(0, 0, 0, 0);
+    background-image: none;
+}
+
+.l-container.l-container--two-column-dashboard .left_column {
+    background-color: rgba(1,1,1,0);
+}
+
+.l-container.l-container--two-column .l-content, .l-container.l-container--two-column-dashboard .l-content {
+    background-color: rgba(1,1,1,0);
+}
+
+.post {
+	background: blue;
+	border-radius: 20px;
```

Comment:
`background: blue` makes the new post container a pretty harsh blue with the title, body, body and tags a white that sticks out.  The new post container also has white corners made visible by `border-radius` that can be fixed with
```css
.post-container {
    background: transparent;
}
```
![image](https://cloud.githubusercontent.com/assets/5816532/10421387/bd58dee8-7075-11e5-9198-6206691d5bf6.png)